apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryResults.cs

```diff
@@ -192,5 +260,58 @@ public BigQueryResults ThrowOnAnyError()
             }
             return this;
         }
+
+        private sealed class AsyncRowEnumerable : IAsyncEnumerable<BigQueryRow>
+        {
+            private readonly BigQueryResults _parent;
+
+            public AsyncRowEnumerable(BigQueryResults parent) => _parent = parent;
+
+            public IAsyncEnumerator<BigQueryRow> GetEnumerator() => new AsyncRowEnumerator(_parent);
+        }
+
+        private sealed class AsyncRowEnumerator : IAsyncEnumerator<BigQueryRow>
+        {
+            private readonly GetQueryResultsOptions _options;
+            private readonly BigQueryResults _parent;
+            private IEnumerator<BigQueryRow> _underlyingIterator;
```

Comment:
nit: rename to `_underlyingEnumerator`?

Reply:
I guess. I prefer the term "iterator" over "enumerator", but that's just a personal thing. Done.